### PR TITLE
The test for iscsi boot fails on somes arches

### PR DESCRIPTION
**Pull request by jbtrystram**
let's disable while we build images for all the arches and create a repo centralizing images that kola needs for the tests.

### CODE REVIEW

File: kola-denylist.yaml [hunk 1]
```diff
@@ -23,3 +23,13 @@
   warn: true
   arches:
     - aarch64
+    - ppc64le
```
Comment (c4rt0) [inline kola-denylist.yaml]: AFAIK the [`coreos.unique.boot.failure`](https://github.com/coreos/coreos-assembler/issues/3669) is now fixed for aarch64. Did you come across issues on ppc64le? or are you referring here to [`coreos.ignition.failure`](https://github.com/coreos/coreos-assembler/issues/3670)?

File: kola-denylist.yaml [hunk 2]
```diff
@@ -23,3 +23,13 @@
...
+  streams:
+    - rawhide
+- pattern: pxe-*.ppcfw
+  tracker: https://github.com/coreos/coreos-assembler/issues/3370
+  # nb: testiso doesn't read this, so it's just for consistency
+  arches:
+    - ppc64le
+- pattern: iso-offline-install-iscsi.bios
+  tracker: https://github.com/coreos/fedora-coreos-tracker/issues/1638
```
Comment (c4rt0) [inline kola-denylist.yaml:27]: Can we maybe snooze it for a month?
snooze: 2024-01-20

File: kola-denylist.yaml [hunk 3]
```diff
@@ -23,3 +23,13 @@
   warn: true
   arches:
     - aarch64
+    - ppc64le
+  streams:
+    - rawhide
+- pattern: pxe-*.ppcfw
```
Comment (travier) [inline kola-denylist.yaml]: I don't understand the diff here. Is this related to this change?

File: kola-denylist.yaml (hunk 1 above)
Comment (travier) [inline kola-denylist.yaml]: same here

**Review by travier — CHANGES_REQUESTED** [review]

File: kola-denylist.yaml (hunk 3 above)
Comment (c4rt0) [inline kola-denylist.yaml]: Due to a lack of knowledge in this area, I just wonder: what is this referring to?

**Comment by jbtrystram** [comment]
@travier @c4rt0  sorry, I messed up the rebase. I added the iscsi pattern but had an old branch, and I got confused when rebasing

**Review by c4rt0 — APPROVED** [review]
/lgtm

**Review by travier — APPROVED** [review]
